<?xml version="1.0"?>

<launch>
  <node pkg="my_robot" type="imu_node" name="imu_node">
  </node>

  <node pkg="my_robot" type="laser_node" name="laser_node">
  </node>

  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se" clear_params="true">
    <rosparam command="load" file="$(find robot_localization)/params/ekf_template.yaml" />
  </node>
  

    <!--  Placeholder for output topic remapping
<remap from="odometry/filtered" to=""/>
  <remap from="accel/filtered" to=""/>
  <!-- </node> -->
<!-- <node pkg="robot_localization" type="ekf_localization_node" name="ekf_se" clear_params="true"> -->
    <!-- <rosparam command="load" file="$(find robot_localization)/params/ekf_template.yaml" /> -->

    <!--  Placeholder for output topic remapping
    <remap from="odometry/filtered" to=""/>
    <remap from="accel/filtered" to=""/>
    -->

  <!-- </node> -->
</launch>
